src/renderer/services/dex.js

```diff
@@ -342,7 +342,7 @@ export default {
               resolve(orders);
             } else {
               // get the next page
-              this.fetchOrderHistory(orders[orders.length - 1].created)
+              this.fetchOrderHistory(orders[orders.length - 1].created, after, sort)
```

Comment:
after is not being updated if there is more than 1 page when using ascending sort order.